htdocs/info/shipyard.php

```diff
@@ -254,8 +254,11 @@ function shipRanking($arr)
 			echo "<tr><td class=\"tbldata\" colspan=\"4\" style=\"text-align:center\">";
 			echo "<div id=\"reqInfo\" style=\"width:100%;\">
 			<br/><div class=\"loadingMsg\">Bitte warten...</div>
-			</div>";	
-			echo '<script type="text/javascript">xajax_reqInfo('.$arr['ship_id'].',"s")</script>';
+			</div>";
+			$shipCatId = $arr['ship_cat_id'];
+			$shipCatIdAlliance = 6;
+			$shipCatAbbr = $shipCatId == $shipCatIdAlliance ? "sa" : "s";
```

Comment:
```suggestion
			$shipCatAbbr = $arr['ship_alliance_shipyard_level'] > 0 ? "sa" : "s";
```
In other place like the alliance shipyard we check the shipyard level